freebsd/FreeBSDProcessList.c

```diff
@@ -494,10 +494,6 @@ void ProcessList_goThroughEntries(ProcessList* this) {
                proc->percent_cpu = proc->percent_cpu / (double) cpus;
          }
       }
-      if (isIdleProcess == false && proc->percent_cpu >= 99.8) {
-         // don't break formatting
-         proc->percent_cpu = 99.8;
-      }
```

Comment:
Hi, thank you for the PR!! Could you explain what situation is this test protecting against? Percentages above 100% shouldn't in principle be a problem for formatting.